### DISCUSSION TITLE
Gettin rid of outline chrome

### DISCUSSION THREAD

**Discussion by jsismail** [vincode-io/Zavala#239] [Support]
Is there a way to toggle off/on the outline chrome?  Looking a clean, indent-only visual.

**Comment by vincode-io** [comment]
There isn't currently a way to hide the toolbar.

**Reply by jsismail** [reply]
I was actually referring to the arrows and dots at the start of each level of the outline.  The veritcal lines that are drawn on the left as well. Is there a way to hide those?

**Reply by vincode-io** [reply]
No, those are always there.

**Reply by jsismail** [reply]
Feature request incoming!

**Reply by vincode-io** [reply]
Please be detailed when you make your request. Given that the disclosure arrow and dots are providing feedback as to if a row is collapsed/expanded or a leaf, they are pretty critical to a functioning outliner. I'm not likely to implement a feature that could accidentally be toggled on and cause a bunch of confusion. You will need to make a strong case as to how this will make Zavala better for a non-significant number of users.

**Reply by jsismail** [reply]
I get it, and that's totally reasonable.  In fact, I wish I had to the cycles to work on a prototype in a branch, which might go a long way in understanding the benefit of such a feature.  All that being said, how exactly do I submit a feature request.  Apologies as I'm new to github.com.

**Reply by jsismail** [reply]
Oh never mind.  I see feature requests under "Issues".  I'll submit there.

**Reply by jsismail** [reply]
Issue submitted: https://github.com/vincode-io/Zavala/issues/241#issue-2329257685

**Reply by smellythief** [reply]
> the disclosure arrow and dots are providing feedback as to if a row is collapsed/expanded or a leaf, they are pretty critical to a functioning outliner.

I agree with this, but what about just toggling off the vertical lines? These add a bit of visual clutter, though I certainly see how they can be useful. I think I'd prefer to work without them for a cleaner look though. What do you think about that? I definitely agree with keeping the arrows and dots though!
